lib/prediction/predictor.js

```diff
@@ -63,6 +63,8 @@ class Worker extends events.EventEmitter {
         ];
         if (process.env.GENIENLP_EMBEDDINGS)
             args.push('--embeddings', process.env.GENIENLP_EMBEDDINGS);
+        if (process.env.DATABASE)
+            args.push('--entity_database', process.env.ENTITY_DATABASE);
```

Comment:
Uhm no? The if statement is wrong. Also, when I say prefixed, I mean prefixed with GENIENLP.